Python/sort/Sequential_MergeSort.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+""" Solution: Sort an array of random integers with merge sort using sequential way."""
+
+import random
+
+""" sequential implementation of merge sort """
+def seq_mergesort(array, *args):
```

Comment:
update `def seq_mergesort(array, *args):` to `def sequential_mergesort(array, *args):`